jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -752,6 +752,13 @@ class PointerObjectConstraintSet final
   [[nodiscard]] const std::vector<ConstraintVariant> &
   GetConstraints() const noexcept;
 
+  /**
+   * Creates a subset graph containing all PointerObjects, their current points-to sets,
+   * and edges representing the current set of constraints.
+   */
+  jlm::util::Graph &
+  DrawSubsetGraph(jlm::util::GraphWriter & writer) const;
```

Comment:
Could we get a unit test on this method? That is one of the big benefits of the internal dot representation. Let's make use of it.